app/views/editions/_edition_import_form.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_with :html => {:class => "form-horizontal"}, :data => {:toggle => "validator"} do |f| %>
+	<div class="col-md-12">
+		<h3>Steam</h3>
+		<h6>Paste the game URL:</h6>
+		<div class="form-group has-feedback">
+			<%= f.label :url, "URL:", :class => "col-sm-3 control-label" %>
```

Comment:
I think this label and the text field should be the same, otherwise if you click on the label, the browser will not put the focus on the field

Reply:
Fixed